board.py

```diff
@@ -5,108 +5,9 @@
-    "BDMICRO VINA-D51 with samd51n20": "bdmicro_vina_d51",
-    "Capable Robot Programmable USB Hub with samd51g19": "capablerobot_usbhub",
-    "Adafruit PyGamer with samd51j19": "pygamer",
-    "DynOSSAT-EDU-EPS with samd21g18": "dynossat_edu_eps",
-    "XinaBox CC03 with samd21g18": "xinabox_cc03",
-    "Adafruit ItsyBitsy M4 Express with samd51g19": "itsybitsy_m4_express",
-    "Seeeduino XIAO with samd21g18": "seeeduino_xiao",
-    "Adafruit Trinket M0 with samd21e18": "trinket_m0",
-    "Adafruit Feather M0 RFM9x with samd21g18": "feather_m0_rfm9x",
-    "J&J Studios datum-Light with None": "datum_light",
-    "Adafruit Feather RadioFruit Zigbee with samr21g18": "feather_radiofruit_zigbee",
-    "BDMICRO VINA-D21 with samd21g18": "bdmicro_vina_d21",
-    "Adafruit Feather M0 Adalogger with samd21g18": "feather_m0_adalogger",
-    "Fluff M0 with samd21e18": "fluff_m0",
-    "Adafruit Metro M4 Express with samd51j19": "metro_m4_express",
-    "Adafruit Metro M4 Airlift Lite with samd51j19": "metro_m4_airlift_lite",
-    "J&J Studios datum-Weather with None": "datum_weather",
-    "Adafruit pIRKey M0 with samd21e18": "pirkey_m0",
-    "Adafruit ItsyBitsy M0 Express with samd21g18": "itsybitsy_m0_express",
-    "HalloWing M0 Express with samd21g18": "hallowing_m0_express",
-    "AloriumTech Evo M51 with samd51p19": "aloriumtech_evo_m51",
-    "UARTLogger II with samd51j19": "uartlogger2",
-    "Trinket M0 Haxpress with samd21e18": "trinket_m0_haxpress",
-    "Adafruit QT Py M0 Haxpress with samd21e18": "qtpy_m0_haxpress",
-    "Arduino Nano 33 IoT with samd21g18": "arduino_nano_33_iot",
-    "CircuitBrains Basic with samd21e18": "circuitbrains_basic_m0",
-    "Winterbloom Sol with samd51j20": "winterbloom_sol",
-    "Escornabot Makech with samd21g18": "escornabot_makech",
-    "Adafruit Hallowing M4 Express with samd51j19": "hallowing_m4_express",
-    "Electronic Cats CatWAN USBStick with samd21e18": "catwan_usbstick",
-    "SAM32v26 with samd51j20": "sam32",
-    "Electronic Cats Bast Pro Mini M0 with samd21e18": "bast_pro_mini_m0",
-    "PicoPlanet with samd21e18": "picoplanet",
-    "Adafruit CircuitPlayground Express with samd21g18": "circuitplayground_express",
-    "Adafruit PyRuler with samd21e18": "pyruler",
-    "Seeeduino Wio Terminal with samd51p19": "seeeduino_wio_terminal",
-    "PyCubedv04-MRAM with samd51j19": "pycubed_mram",
-    "Adafruit Pybadge Airlift with samd51j20": "pybadge_airlift",
-    "Hacked Feather M0 Express with 8Mbyte SPI flash with samd21g18": "feather_m0_supersized",
-    "Adafruit Pybadge with samd51j19": "pybadge",
-    "Mini SAM M4 with samd51g19": "mini_sam_m4",
-    "uChip with samd21e18": "uchip",
-    "Cedar Grove StringCar M0 Express with samd21e18": "stringcar_m0_express",
-    "CircuitBrains Deluxe with samd51j19": "circuitbrains_deluxe_m4",
-    "uGame10 with samd21e18": "ugame10",
-}
+board_id_map_file = open("board_id_map.json", "r")
```

Comment:
Can you switch these to `with open()....`? It makes a cleaner resource cleanup pattern (you can skip the close)

Reply:
Yep, done with latest commit.